astropy/table/operations.py

```diff
@@ -129,6 +130,98 @@ def join(left, right, keys=None, join_type='inner',
     return out
 
 
+def setdiff(table1, table2, keys=None):
+    """
+    Take a set difference of table rows.
+
+    The row set difference will contain all rows in ``table1`` that are not
+    present in ``table2``. If the keys parameter is not defined, all columns in
+    ``table1`` will be included in the output table.
+
+    Parameters
+    ----------
+    table1 : Table object
```

Comment:
Replace 'Table object' by '`~astropy.table.Table`' (in backticks) - same for table2 and keys

Reply:
@astrofrog - I don't understand the comment for `keys` (it's not a `Table`).

Reply:
Woops ignore ``keys`` :)